controllers/fcmToken.js

```diff
@@ -6,15 +6,15 @@ const { saveFcmToken } = require("../models/fcmToken");
  * @param req {Object} - Express request object
  * @param res {Object} - Express response object
  */
-const fcmTokenController = (req, res) => {
+const fcmTokenController = async (req, res) => {
   const { fcmToken } = req.body;
 
   // Save the FCM token to your database or perform other necessary actions.
   // You can associate the token with a user or device, for example.
 
-  const fcmTokenId = saveFcmToken({ userId: req.userData.id, fcmToken });
-
-  res.status(200).send(`FCM token saved: ${fcmTokenId}`);
+  const fcmTokenId = await saveFcmToken({ userId: req.userData.id, fcmToken });
+  if (fcmTokenId) res.status(200).json({ status: 200, message: "Device registered successfully" });
+  else res.status(409).json({ status: 409, message: "Device Already Registered" });
```

Comment:
can we incorporate `try catch block`

Reply:
it was in refactor-notify did that please check once again